sdk/ai/Azure.AI.Agents.Persistent/tests/PersistentAgentsTests.cs

```diff
@@ -2306,6 +2306,58 @@ await ValidateStream(
                 );
         }
 
+        [RecordedTest]
+        [TestCase(true, "adani")]
+        //TODO: The Image URI is not supported, uncomment this text when the ICM 686545924 will be resolved.
+        //[TestCase(false, "trail")]
+        public async Task TestImageAsInput(bool useUploaded, string expectedWord)
```

Comment:
Was this meant to be commented out? From the test seems like you wanted this test to be disabled for now.

Reply:
Yes, I have created the ICM for the service team. We can use image as an input, however, using URL will result in 500 error, when we will start the run.